src/PJ_labrd.c

```diff
@@ -46,7 +46,7 @@ static XY e_forward (LP lp, PJ *P) {          /* Ellipsoidal, forward */
 static LP e_inverse (XY xy, PJ *P) {          /* Ellipsoidal, inverse */
     LP lp = {0.0,0.0};
     struct pj_opaque *Q = P->opaque;
-    double x2, y2, V1, V2, V3, V4, t, t2, ps, pe, tpe, s;
+    double x2, y2, V1, V2, V3, V4, t = 0.0, t2, ps, pe, tpe, s;
```

Comment:
I recommend tagging (with a comment) any unnecessary initializations performed to suppress a warning.  Then a developer 5 years from now won't be left wondering whether the value t = 0.0 ever gets used.